.env

```diff
@@ -0,0 +1,21 @@
+#Postgres environment variable
+
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=um_db
+UM_PASSWORD=gateway
+
+
+#User-Manager environment variable
+
+SERVICE_NAME=usermanager
+BIND_IP=0.0.0.0
+BIND_PORT=8000
+BIND_DEBUG_PORT=8001
+CONSUL_ADDRESS=consul:8500
+
+#Gralog environment variable
+GRAYLOG_PASSWORD_SECRET=usermanagerlvl48
+# Password: gopher
+GRAYLOG_ROOT_PASSWORD_SHA2=9cc1ee455a3363ffc504f40006f70d0c8276648a5d3eb3f9524e94d1b7a83aef
+GRAYLOG_HTTP_EXTERNAL_URI=http://127.0.0.1:9000/
```

Comment:
It is not a good idea to push files like this.
Add .env to .gitignore

Reply:
Fix problem